tests/remote_js/api3.js

```diff
@@ -0,0 +1,933 @@
+const HTTP = require("./httpHandler");
+const { JSDOM } = require("jsdom");
+const API2 = require("./api2.js");
+const Helpers = require("./helpers");
+const fs = require("fs");
+const wgxpath = require('wgxpath');
+
+class API3 extends API2 {
```

Comment:
Right, I did the extension to avoid duplicate code at first but since it is not an issue, I made independent api2/3 and helpers2/3 that have some functions overlapping